### PR TITLE
Pushing prow pod utils images using ko

### PR DESCRIPTION
**Pull request by chaodaiG**
Local testing looks promising.

/hold
Depends on: https://github.com/kubernetes/test-infra/pull/25154

/cc @BenTheElder @cjwagner

### CODE REVIEW

File: prow/prow-images.sh [hunk 1]
```diff
@@ -21,7 +21,38 @@ REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
...
+# takes comma separated list of arch
+# returns space separated list of tags basesd on arch
+tags-arg() {
+  local tags=(${TAG_SET[@]})
+  local arches="$1"
+  if [[ "${arches}" == "all" ]]; then
+    for arch in "${ALL_ARCHES[@]}"; do
+      tags+=("${arch}")
+      for base in "${TAG_SET[@]}"; do
+        tags+=("${arch}-${base}")
```
Comment (cjwagner) [inline prow/prow-images.sh:46]: This is backwards it should be base-arch.

File: prow/prow-images.sh [hunk 2]
```diff
@@ -21,7 +21,38 @@ REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
...
+  "latest"
+  "latest-root"
+  "${GIT_TAG}"
+)
+ALL_ARCHES=("arm64" "s390x" "ppc64le")
+
+# push or local tar?
+PUSH="${PUSH:-false}"
+
+# takes comma separated list of arch
```
Comment (cjwagner) [inline prow/prow-images.sh:37]: This isn't accurate.

File: prow/.prow-images [hunk 3]
```diff
@@ -1,2 +1,5 @@
-prow/cmd/deck
```
Comment (cjwagner) [inline prow/.prow-images:1]: This looks like an unintentional removal?

**Comment by chaodaiG** [comment]
#25157  contains very similar changes in `prow/prow-images.sh`, and advanced further than this PR, addressed reviews over there, please review that one instead

/hold

**Comment by chaodaiG** [comment]
/close
will open a new PR

**Pull request closed without merging**